node/overseer/src/lib.rs

```diff
@@ -1134,7 +1134,7 @@ impl metrics::Metrics for Metrics {
 				prometheus::Histogram::with_opts(
 					prometheus::HistogramOpts {
 						common_opts: prometheus::Opts::new(
-							"parachain_channel_fill_level_from_overseer",
+							"overseer_channel_fill_level_from_overseer",
```

Comment:
is there a reason to use a different prefix for overseer? we already mention its name at the end
also prometheus recomments to use a suffix describing the unit:
https://prometheus.io/docs/practices/naming/

Reply:
@ordian If you add a change to the name, feel free to suggest one. I added overseer since there already was one and it seemed more fitting than `parachain`.